lib/pharos/resources/calico/20-configmap.yml.erb

```diff
@@ -7,6 +7,15 @@ metadata:
 data:
   ipv4_pool_cidr: "<%= ipv4_pool_cidr %>"
 
+
+
+  # Configure the Calico backend to use.
+  calico_backend: "bird"
+
+  # TODO Do we need mtu setting?
+  # Configure the MTU to use
+  veth_mtu: "1440"
```

Comment:
How is this mtu different vs cni_network_config mtu?

Reply:
By accident, good catch :) Fixed now.

Reply:
Ok, but why do we need both?

Reply:
> When using Kubernetes self-hosted manifests, the CNI plugin derives the MTU value from the veth_mtu field of the calico-config ConfigMap, and it is set to 1440 by default. On restart of the calico/node pods, any references to `__CNI_MTU__` are replaced by the veth_mtu value and inserted into the CNI configuration file (aka conflist) at the directory specified by Kubernetes (currently defaults to /etc/cni/net.d/).

Reply:
Their reference manifests have it in both places so didn't really dare to remove.